### PR TITLE
io-throttling:Add throttle_runtime interface

### PR DESCRIPTION
**Pull request by zhenyzha**
Add throttle_runtime interface

ID:2026574
Signed-off-by: Zhenyu Zhang <zhenyzha@redhat.com>

### CODE REVIEW

**Comment by zhenyzha** [comment]
` (1/1) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.6.0.ppc64le.io-github-autotest-qemu.throttle_parameter_test.normal_rw.with_iops_bps_rw: PASS (774.66 s)`

@qingwangrh @YongxueHong Could you pls help review when you are free. thanks.

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=throttle_operation_test.group_change.to_burst --guestname=RHEL.8.4.0 --machines=q35 --driveformat=virtio_scsi --clone=no --debug --customsparams="extra_params = -global ICH9-LPC.acpi-pci-hotplug-with-bridge-support=off"

(1/1) Host_RHEL.m8.u6.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.throttle_operation_test.group_change.to_burst.q35: PASS (367.48 s)

LGTM

File: qemu/tests/cfg/throttle_parameter_test.cfg [hunk 1]
```diff
@@ -71,6 +71,8 @@
           - with_iops_bps_rw:
               group1 = {"bps-read":163840,"iops-read":20,"bps-write":163840,"iops-write":20}
               group2 = {"bps-read":204800,"iops-read":25,"bps-write":204800,"iops-write":25}
+              ppc64,ppc64le:
+                throttle_runtime = 300
```
[the remark this answers is not in the document]
Comment (YongxueHong) [reply qemu/tests/cfg/throttle_parameter_test.cfg:75]: I think that it is also better to set the `throttle_runtime` for the different throttle groups.
```
runtime_throttle_group1= 300
runtime_throttle_group2= 100
```
This is just my thought, depending on your requirements.

[the remark this answers is not in the document]
Comment (zhenyzha) [reply qemu/tests/cfg/throttle_parameter_test.cfg]: thanks a lot, corrected.
The current case is to test the difference between different throttles at the same time, so there is no need for grouping time.
If there is a need for optimization in the future, maybe we can optimize it in another patch.

[the remark this answers is not in the document]
Comment (YongxueHong) [reply qemu/tests/cfg/throttle_parameter_test.cfg]: Okay.
LGTM.

**Pull request merged**